lms/templates/commerce/provider.underscore

```diff
@@ -1,27 +1,26 @@
 <div class="provider-wrapper">
   <div class="provider-info">
-    <%= interpolate(
-          gettext("You still need to visit the %(display_name)s website to complete the credit process."),
-          { display_name: display_name }, true
+    <%- HtmlUtils.interpolateHtml(
```

Comment:
You can replace `HtmlUtils.interpolateHtml` with `StringUtils.interpolate` here, otherwise you'd be double-escaping.